homework/insensitive-palindrom/palindrome.cpp

```diff
@@ -0,0 +1,11 @@
+#include "palindrome.hpp"
+#include <algorithm>
+#include <cctype>
+
+bool is_palindrome(std::string text) {
+    text.erase(std::remove_if(text.begin(), text.end(), [](unsigned char c)
+                              { return isspace(c) || !isalpha(c); }),
+               text.end());

```

Comment:
I'm thinking will be more optimized if you put this requirement in transform, or from transform to remove_if

Reply:
The is no sth like more optimized ;p You can failed engeenering exam if you say this :D

But backing to project: This is good solution, is better to remove uneccessary character before convert them to lower (we will have less elements).
Ofc you can wite wn function remove_tranform_if and then reject unecessary elements and transform rest of them. But generally this will not have a big impact to time